src/ribbon/panel.cpp

```diff
@@ -711,7 +711,9 @@ bool wxRibbonPanel::Realize()
                 scale = 2.0;
 
             wxImage img(m_minimised_icon.ConvertToImage());
-            img.Rescale(int(scale * bitmap_size.GetWidth()), int(scale * bitmap_size.GetHeight()), wxIMAGE_QUALITY_HIGH);
+            img.Rescale(static_cast<int>(scale * bitmap_size.GetWidth()),
```

Comment:
I wonder if this shouldn't be `wxRound()`...

Reply:
Will make the change.